src/ray/raylet/task_dependency_manager.h

```diff
@@ -126,7 +131,9 @@ class TaskDependencyManager {
   struct TaskDependencies {
     /// The objects that the task is dependent on. These must be local before
     /// the task is ready to execute.
-    std::unordered_set<ObjectID> object_dependencies;
+    // std::unordered_set<ObjectID> object_dependencies;
```

Comment:
Remove this line and update the documentation for `get_dependencies` and `wait_dependencies`.